aaudio/echo/src/main/cpp/echo_audio_engine.cc

```diff
@@ -123,11 +123,13 @@ void EchoAudioEngine::stopStreams() {
   if (recordingStream_ != nullptr) {
     stopStream(recordingStream_);
     closeStream(recordingStream_);
+    recordingStream_ = nullptr;
   }
 
   if (playStream_ != nullptr) {
     stopStream(playStream_);
     closeStream(playStream_);
+    playStream_ = nullptr;
```

Comment:
added more comment to closeStream() function: pointing out the fact that AAudioStream_close() already takes good care of the synchronization between datacallback and close() function.  PTAL, thanks